project/JsBuild.scala

```diff
@@ -15,6 +15,7 @@ import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 trait JsBuild { this: BuildCommons =>
 
   val scalaJSVersion = Option(System.getenv("SCALAJS_VERSION")).getOrElse("1.0.0")
+  val sjsPrefix = if (scalaJSVersion.startsWith("1.0.")) "_sjs1.0_" else "_sjs0.6_"
```

Comment:
It's actually `"_sjs1_"` for 1.x.

Reply:
And it will stay like that for all 1.x version, so the test should be `startsWith("1.")`, not `startsWith("1.0.")`.

Reply:
@sjrd Adjusted in the following commit: 

https://github.com/scalatest/scalatest/pull/1770/commits/4d6e31855a9c223e0534e33b948c0549e0f8a6fd

Thanks for catching it!